.github/workflows/regression-test-auto.yml

```diff
@@ -0,0 +1,76 @@
+name: Regression Tests - Agglayer
+'on':
+  pull_request:
+    types: [opened, synchronize]  # Trigger on new pull requests and when existing ones are synchronized
+    branches:
+      - main
+    inputs:
+      bake_time:
+        description: bake time (minutes)
+        required: false
+        default: 20
+jobs:
+  deploy_devnet:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Docker
+        uses: docker/setup-buildx-action@v1
+      - name: Clone and build agglayer image locally
+        run: |
+          git clone https://github.com/0xPolygon/agglayer.git
```

Comment:
Is this really needed if the current repo is already agglayer?

Reply:
> Is this really needed if the current repo is already agglayer?

good call! and I think we should build the Docker image using  `Dockerfile.release`.

https://github.com/0xPolygon/kurtosis-cdk/blob/main/.github/workflows/regression-tests.yml#L52-L56

Reply:
fixed, good call